git-quick-stats

```diff
@@ -274,18 +276,19 @@ function showMenu() {
     printf %b "${NUMS}    8)${TEXT} Branch tree view (last $_limit)\\n"
     printf %b "${NUMS}    9)${TEXT} All branches (sorted by most recent commit)\\n"
     printf %b "${NUMS}   10)${TEXT} All contributors (sorted by name)\\n"
-    printf %b "${NUMS}   11)${TEXT} Git commits per author\\n"
-    printf %b "${NUMS}   12)${TEXT} Git commits per date\\n"
-    printf %b "${NUMS}   13)${TEXT} Git commits per month\\n"
-    printf %b "${NUMS}   14)${TEXT} Git commits per year\\n"
-    printf %b "${NUMS}   15)${TEXT} Git commits per weekday\\n"
-    printf %b "${NUMS}   16)${TEXT} Git commits per weekday by author\\n"
-    printf %b "${NUMS}   17)${TEXT} Git commits per hour\\n"
-    printf %b "${NUMS}   18)${TEXT} Git commits per hour by author\\n"
-    printf %b "${NUMS}   19)${TEXT} Git commits per timezone\\n"
-    printf %b "${NUMS}   20)${TEXT} Git commits per timezone by author\\n"
+    printf %b "${NUMS}   11)${TEXT} New contributors\\n"
```

Comment:
Do we want to add how this is sorted here? For example, I think it's sorted by email right now (`sort` being called on the `%aE` format string). Do we want that? Should we sort it by most recent commit? Either way, I think mentioning how this is sorted after this description would be helpful to users.

Reply:
Sorting by email makes more sense to how I intended to use this, which is to recognize new contributors each quarter. I agree adding an induction of how it's sorted is good, so I'll do that. If someone decides a different sort order makes more sense, I won't object.